boxes/__init__.py

```diff
@@ -218,8 +218,8 @@ def html(self, name, default, _):
         if isinstance(default, (str)):
             default = self(default)
         return """<input name="%s" type="hidden" value="0">
-<input name="%s" type="checkbox" value="1"%s>""" % \
-            (name, name, ' checked="checked"' if default else "")
+<input name="%s" id="%s" aria-labeledby="%s %s" type="checkbox" value="1"%s>""" % \
+            (name, name, name, name+"_id", name+"_description",' checked="checked"' if default else "") #fix here
```

Comment:
adds associated of label to the left of the input box and description to the right to the input box so screen readers will announce items it does this with an id tag and aria-labeledby tags that reference labelfor tags and id tags there is some redundancy with the aria-labeledby tag referencing the same tag as the label for tag (ie name_id) this is to provide robustness across different screen readers.